.github/ISSUE_TEMPLATE/bug.yml

```
@@ -22,6 +22,8 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I searched for similar issues at https://github.com/Homebrew/homebrew-core/issues and found no duplicates.
```

> But it usually won't improve situations with closed issues unless we direct users to search all open+closed issues

Then maybe we should make it more explicit:

```suggestion
        - label: I searched for similar open and closed issues at https://github.com/Homebrew/homebrew-core/issues and found no duplicates.
```

> it's a good candidate to either pin it or keep it open.

I think we might need a better reason to pin an issue that's actually not related to us.

I mean, I don't actually expect this to stop users from opening duplicate issues ever again, either. What I do hope is to at least make this a requirement for opening an issue. (It's already required in our discussion board.) And that should help us to reduce the number of duplicate issues, more or less.

---

`https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue` will show you open and closed issues

I think this might reduce duplicates but: given we have >5000 issues there we might want to scope it to something like "on the first page" so it's not an unreasonable amount?

---

> I think this might reduce duplicates but: given we have >5000 issues there we might want to scope it to something like "on the first page" so it's not an unreasonable amount?

I think the search range should usually be narrowed down significantly with some search keywords. (But yeah, I understand that some issues are indeed hard to search.) Anyway, it's not the point to say "we'll blame you for not being able to find the duplicate issue opened two years ago".